core-data/resolvers.js

```diff
@@ -33,16 +43,37 @@ export async function* getAuthors() {
 	yield receiveUserQuery( 'authors', users );
 }
 
+async function* loadKindEntities( state, kind ) {
```

Comment:
This isn't so much a resolver, is it? More a utility?

Reply:
Well, the difference is not totally clear here since it uses selectors and yield actions.
Anyway, I moved it to the `entities.js` file for now.